### PR TITLE
ENH: natural sorting, fixes #831

### LINKED ISSUE #831: Samples column should not be sorted lexicographically
In the study results table (such as http://qiita.colorado.edu/study/public/), the Samples column reports the number of samples in the study. It currently sorts lexicographically, but this is not very useful. It should sort numerically.

### PR DESCRIPTION
**Pull request by wasade**
I only have 1 study in test, and my demo database is hosed, so I can't reallllly verify this one.

### CODE REVIEW

File: qiita_pet/templates/private_studies.html [hunk 1]
```diff
@@ -3,13 +3,18 @@
...
 
 $(document).ready(function() {
-    $('#user-studies-table').dataTable({"order": [[ 1, "asc" ]]});
-    $('#shared-studies-table').dataTable({"order": [[ 1, "asc" ]]});
+        $('#user-studies-table').dataTable({
+            order: [[ 1, "asc" ]],
+            columnDefs: 'natural'});
+        $('#shared-studies-table').dataTable({
+            order: [[ 1, "asc" ]],
+            columnDefs: 'natural'});
```
Comment (adamrp) [inline qiita_pet/templates/private_studies.html]: haven't tested this yet, but the code here differs slightly from the example in natural.js

I'll test shortly.

Comment (wasade) [inline qiita_pet/templates/private_studies.html]: you're right... not sure what i was thinking. lemme know if it works.

Comment (adamrp) [inline qiita_pet/templates/private_studies.html]: Nope, I don't think it works see screenshot below of studies sorted by number of samples ascending):

image

Note that the problem only presents when the column is a mixed datatype (integers and strings in the column, as is the case when there are some studies that have `num_samples_collected` set to a number, and some studies that do not have it set (which displays as `None`)

Comment (adamrp) [inline qiita_pet/templates/private_studies.html]: Even when I change the code to reflect the example, it doesn't work. There's also not great documentation on this plugin, which doesn't help...

Comment (wasade) [inline qiita_pet/templates/private_studies.html]: No... and I can't test locally as my demo database is hosed

Comment (adamrp) [inline qiita_pet/templates/private_studies.html]: Can you drop your environment (`qiita_env drop`) and create a new test environment (`qiita_env make --no-load-ontologies` with TEST_ENVIRONMENT=TRUE in your qiita_config)? If you do that, then create two studies on the test@foo.bar user, then change `number_samples_collected` on one of the studies using psql to, e.g., 210, you should be able to replicate the test I'm running here

Comment (adamrp) [inline qiita_pet/templates/private_studies.html]: Got it -- didn't mean to duplicate effort on this, but your being unable to test led me to tinkering :smile:

I'll submit a PR against your fork

**Comment by adamrp** [comment]
@wasade, when I was trying to do a PR against your fork, it kept pulling in a bunch of commits, so I opened a new PR; I think this one can be closed, since the other one contains the commits from this PR

**Pull request closed without merging**